### ISSUE TITLE
Not working on linux

### ISSUE THREAD

**Issue by abu-cwarky** [200MPH/tnt#27]
Hello again. I moved my project from windows to linux and I'm stuck.
On windows it's working perfect (IIS and php 7.4) but on linux (apache and php7.4 - ubuntu) I'm getting this : "Array ( [0] => Response XML document is not valid or empty ) ". I checked XML and seems to be fine.
Do you have any ideea what I might check? Thanks.

**Comment by abu-cwarky** [comment]
I think I found the problem : 

Warning: file_get_contents(): SSL operation failed with code 1. OpenSSL Error messages: error:141A318A:SSL routines:tls_process_ske_dhe:dh key too small 

Warning: file_get_contents(): Failed to enable crypto 

Warning: file_get_contents(https://express.tnt.com/expressconnect/shipping/ship): failed to open stream: operation failed

Looking for a solution. Thanks

**Comment by abu-cwarky** [comment]
It seems to be an OpenSSL issue.
I edited openssl.conf and put this in the beggining:
`openssl_conf = default_conf`
ant at the end: 
`[ default_conf ]
ssl_conf = ssl_sect

[ssl_sect]
system_default = ssl_default_sect

[ssl_default_sect]
MinProtocol = TLSv1.2
CipherString = DEFAULT:@SECLEVEL=1`
Problem solved.

**Comment by 200MPH** [comment]
Hey,
have you try $shipping->disableSSLVerify() ?

**Comment by abu-cwarky** [comment]
Hello.
Yeah, and it did not matter the request was never sent because of weak encryption on TNT side. I HAD to lower security because not even cURL was working. I mean linux command not php curl. To make it work I had to specify "--ciphers 'DEFAULT:!DH'". I think there is no other way using file_get_contents. Maybe php cURL have some workarounds but I don't think it worth the effort.
Or at least I did not find any :)
Have a beautiful day :)
